raft/src/test/java/org/apache/kafka/raft/RaftEventSimulationTest.java

```diff
@@ -129,13 +129,16 @@ public void testReplicationNoLeaderChangeQuorumSizeFive() throws IOException {
     }
 
     private void testReplicationNoLeaderChange(QuorumConfig config) throws IOException {
+        Set<Integer> voters = getVoters(config.numVoters);
```

Comment:
Do we need this? `Cluster` already has a method to get the set of voters.

Reply:
Yea, good call!